09-features.md

```diff
@@ -39,6 +39,11 @@ The Context column decodes as follows:
 | 16/17 | `basic_mpp`                      | Node can receive basic multi-part payments                | IN9      | `payment_secret`  | [BOLT #4][bolt04-mpp]                 |
 | 18/19 | `option_support_large_channel`   | Can create large channels                                 | IN       |                   | [BOLT #2](02-peer-protocol.md#the-open_channel-message) |
 | 20/21 | `option_anchor_outputs`          | Anchor outputs                                            | IN       | `option_static_remotekey` | [BOLT #3](03-transactions.md)         |
+| 22/23 | `option_anchors_zero_fee_htlc_tx` | Anchor commitment type with zero fee HTLC txs             | IN       |                   | [BOLT #3][bolt03-htlc-tx], [lightning-dev][ml-zero-fee-vuln]|
```

Comment:
Depends on option_static_remotekey, please.

Reply:
I guess this way deliberate because of https://github.com/lightningnetwork/lightning-rfc/pull/824#issuecomment-742926077 :

> On second thought, maybe it is a good idea to make all commitment formats dependency free...

Reply:
No, it's a terrible idea because it makes it look like you can negotiate a option_anchors_zero_fee_htlc_tx channel without also negotiating `option_static_remotekey` and you'll just fail if you try because we won't be supporting that.  We certainly won't be testing it (and neither will anyone else).